src/feature/dirparse/policy_parse.c

```diff
@@ -201,11 +202,24 @@ router_parse_addr_policy_private(directory_token_t *tok)
   if (!arg)
     return NULL;
 
+  switch (*arg) {
+    case '4':
+      private_flag = ADDR_POLICY_PRIVATE4;
+      break;
+    case '6':
+      private_flag = ADDR_POLICY_PRIVATE6;
+      break;
+    default:
```

Comment:
can this really be any character here?  or are these some characters we don't want to accept?